src/ctap2/commands/client_pin.rs

```diff
@@ -461,6 +467,7 @@ pub enum PinError {
     InvalidPin(Option<u8>),
     PinAuthBlocked,
     PinBlocked,
+    #[serde(skip)] // TODO: Use 'with' here and just convert to string
```

Comment:
Let's do it :)

Reply:
I moved this one level down. Each crypto BackendError now needs to be serializable.